### PR TITLE
[FIX] solved override conflicts for decidim 19

### PR DESCRIPTION
**Pull request by victorol1**
Solved override conflicts.

Related to basecamp's: https://3.basecamp.com/4186608/buckets/13990309/todos/2266764430

### CODE REVIEW

**Comment by victorol1** [comment]
@tramuntanal revisa cuando puedas

File: app/views/decidim/authorization_modals/_content.html.erb [hunk 1]
```diff
@@ -23,14 +23,7 @@
...
         <% status.data[:fields].each do |field, value| %>
-          <% case field %>
-          <% when :birthdate %>
-            <li> <strong> <%= text_invalid_age_authorizer(status) %> </strong> </li>
-          <% when :subcensus %>
-            <li> <strong> <%= t(".subcensus.invalid_field", scope: "diba")%> </strong> </li>
-          <% else %>
-            <li><strong><%= t ".#{status.code}.invalid_field", field: t("#{status.handler_name}.fields.#{field}", scope: "decidim.authorization_handlers"), value: value ? "(#{value})" : "" %></strong></li>
-          <% end %>
+          <li><strong><%= t ".#{status.code}.invalid_field", field: t("#{status.handler_name}.fields.#{field}", scope: "decidim.authorization_handlers"), value: value ? "(#{value})" : "" %></strong></li>
```
Comment (tramuntanal) [inline app/views/decidim/authorization_modals/_content.html.erb]: why are we removing this customization?

File: app/views/decidim/authorization_modals/_content.html.erb [hunk 2]
```diff
@@ -23,14 +23,7 @@
...
       <ul>
         <% status.data[:fields].each do |field, value| %>
-          <% case field %>
-          <% when :birthdate %>
-            <li> <strong> <%= text_invalid_age_authorizer(status) %> </strong> </li>
-          <% when :subcensus %>
-            <li> <strong> <%= t(".subcensus.invalid_field", scope: "diba")%> </strong> </li>
-          <% else %>
-            <li><strong><%= t ".#{status.code}.invalid_field", field: t("#{status.handler_name}.fields.#{field}", scope: "decidim.authorization_handlers"), value: value ? "(#{value})" : "" %></strong></li>
-          <% end %>
```
Comment (tramuntanal) [inline app/views/decidim/authorization_modals/_content.html.erb]: why are we removing this customization?

Comment (victorol1) [reply app/views/decidim/authorization_modals/_content.html.erb]: Mistake, it shouldn't be removed. Reverted changes.

**Pull request merged**